docs/hw/jetson_hookup.md

```diff
@@ -1,32 +1,86 @@
 # iRobot® Create® 3 Hookup Guide for NVIDIA® Jetson™
 
+The iRobot® Create® 3 robot has a JST XH-style connector on its internal [Adapter Board](./adapter.md) which is capable of supplying a **maximum of 2A** of curret at the current battery volrage (14.4V nominal).
+
 The iRobot® Create® 3 robot has a USB-C®[^1] connector implementing a USB 2.0 host capable of 5 V @ 3 A continuous, which can be used to power and communicate with various downstream devices.
+
 Here are some hookup examples for NVIDIA® Jetson™[^2] computers.
 
-## NVIDIA® Jetson Nano™ 2GB
-The Jetson Nano™ 2GB has a USB-C® port (J2) for power and a USB Micro-B port (J13) for downstream data.
-This can be connected to the Create 3 robot most simply using a USB-C® hub and two cables -- USB A to Micro-B and USB A to USB-C®.
-![Hookup diagram for Jetson Nano™](data/hookup_nano2gb.jpg "Jetson Nano™ 2GB")
+## NVIDIA® Jetson Orin Nano™ Developer Kit
+
+!!! note 
+
+    ### Hookup kit
+    
+    Below lists the content of an example kit that enables mounting Jetson Orin Nano Developer Kit with some sensors like Intel®  RealSense™[^3].
+
+    - Jetson Orin Nano Developer Kit
+    - Custom power cable (JST-HX to DC 5.5-2.5mm plug)
```

Comment:
Did you mean JST-XH?

Reply:
[d9abea2](https://github.com/iRobotEducation/create3_docs/pull/385/commits/d9abea206a798e3a143f94938c5fc868c027d05c)